src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -279,6 +386,18 @@
  *   &lt;property name=&quot;allowEmptyCatches&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public static void main(String[] args) {
+ *         try {
+ *             int k = 10 / 0;
+ *             int a=4; // 2 violations , '=' is not followed and preceded by whitespace.
+ *         } catch (Exception e){} // ok
```

Comment:
we need in some othe rexample on above came catch to show violation on it.

Reply:
@romani Do you want me to add this example in the default config too?

Reply:
yes, please, but try to minimize try-catch block to not have extra code.

Reply:
Done @romani